application/template/NGSILD/goTemplate/ngsiclient.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+)
+
+func queryContext(id string, IoTBrokerURL string) (map[string]interface{}, error) {
```

Comment:
[minor] Function specification is missing here. Mention what is the use of this function.

Reply:
Done